### DISCUSSION TITLE
No HD detected on Mac SE

### DISCUSSION THREAD

**Discussion by dspitzley** [ZuluSCSI/ZuluSCSI-firmware#102] [Q&A] [answered]
Hey all! Just attempted to hook up the ZuluSCSI to my Mac SE. I'm running into an issue, unfortunately. I get the flashing "no disk" icon after booting. The debug zululog.txt indicates everything is good. It's possibly a misunderstanding on my part about what sort of pre-generated disk images ZuluSCSI can use. I created a OS 6.0.8 image with Basilisk II (and Mini vMac), and everything seems fine in the emulators, but after naming the image HD0.img and putting it on the SD card, the SE doesn't detect it as a valid hard drive. This seems like something that should work, but I suppose that pre-generated hard drive images may not work the way I expect. Any help would be appreciated.

Thanks,
Dan

**Comment by aperezbios** [comment, marked as answer]
Dan,

It's likely that the image you created doesn't have the necessary Apple Partition Map, and is just a raw bootable HFS filesystem, which no actual Mac can boot from. Emulators can just fine, because the way they expose storage to the OS is as if it were a giant floppy drive/image, not via actual SCSI. All 68K Macintosh ROMs will only boot from an HFS volume that's contained within a block device that contains an Apple Partition Map. 

If you take the image from https://macintoshgarden.org/sites/macintoshgarden.org/files/apps/RaSCSI-Boot-6.0.8.hda__1.zip and copy the contents of your existing image over to it, within an emulator, it should boot fine.

**Reply by dspitzley** [reply]
Gotcha. I'll give it a try. Is there any way to increase the size of that image to 1GB? Its current size of 40MB is too small for my needs.

Thanks,
Dan

**Reply by aperezbios** [reply]
No, but there are plenty of other pre-made images out there of various sizes. Here's one https://www.savagetaylor.com/wp-content/uploads/68k_Macintosh/Bootdisks/608_2GB_drive.zip

And you can also make your own.

**Comment by dspitzley** [comment]
I’ll give that one a try. I’ve been attempting to create my own, but it hasn’t gone well. I generated a 2GB image in Basilisk, brought it over to the SE as HD1 with HD0 being that RaSCSI image you pointed me to. Using Apple HD SC Setup I’ve been trying to initialize HD1 and after spinning for an hour it says it failed due to being unable to mount the drive. I have SW1 enabled.

Dan
